chia/wallet/wallet.py

```diff
@@ -294,6 +291,41 @@ async def select_coins(self, amount, exclude: List[Coin] = None) -> Set[Coin]:
                 "Can't make this transaction at the moment. Waiting for the change from the previous transaction."
             )
 
+        # Use older coins first
+        valid_unspent.sort(reverse=True, key=lambda r: r.coin.amount)
```

Comment:
This is the wrong ordering for the Bitcoin knapsack algo

Reply:
Descending order is the correct order if i am not mistaken.